packages/design-system/src/components/FormLabel/useFormLabel.tsx

```diff
@@ -149,9 +151,7 @@ export function useFormLabel<T extends UseFormLabelProps>(props: T) {
     id,
     errorMessage,
     inversed,
-    'aria-describedby': hasBottomError
```

Comment:
I don't understand what this was doing? It looks like it was only applying `aria-describedby` if a bottomError was set? Why?

Regardless, because this was happening here, I was unable to apply the recommended fix for all form labels. I removed so we could test this solution, but also it seems the `bottomError` placement is broken.